app/classes/mappable/box_methods.rb

```diff
@@ -66,6 +66,21 @@ def east_west_distance
       west <= east ? east - west : east - west + 360
     end
 
+    # Converts degrees to radians.
+    def to_radians(degrees)
+      degrees * Math::PI / 180.0
+    end
+
+    # Arbitrary test for whether box is "vague" (i.e., covers a large area).
+    # Uses formula for the area of a patch of a spherical earth where R = 6400.
+    # Rˆ2 * (long2 - long1) * (sin(lat2) - sin(lat1))
+    def vague?
```

Comment:
Will this ever get called on an "undefined" Location?
If so, will `north_south_distance` and `east_west_distance` throw Errors?

Reply:
I don't believe so.